content/applications/inventory_and_mrp/inventory/management/products/strategies.rst

```diff
@@ -2,75 +2,112 @@
+there is no specific rule for a product, it's assumed that the min and max stock are 0.
 
-The Minimum Stock Rules configuration is available through your
-Inventory module. In the Inventory Control menu select "Reordering Rule"
-in the drop down menu. There, click on "Create" to set minimum and
-maximum stock values for a given product.
-
-.. image:: strategies/strategies01.png
-    :align: center
+.. image:: strategies/replenishment-report.png
+   :align: center
+   :alt: The Replenishment Report shows products that need to be reordered manually.
 
 .. note::
-    Show tooltips for "minimum quantity", "maximum quantity" and "quantity multiple"
-    fields
+   - If a product doesn't have any specific rule, the system will check the forecast in an infinite
+     future.
+   - If a product has a specific rule, the system will check the forecast at D +
+     purchase/manufacturing lead time + security lead time.
+
+A Reordering Rule can be added through the replenishment report by simply adding a line in the
+report. Be sure to have necessary information like supplier or bill of material configured on your
+product.
+
+By default, the quantity to order is the quantity required to fill your max quantity in stock but
+it can be adjusted. To replenish manually, just hit **Order Once**.
+
+Rules can create automatically a Purchase Order or a Manufacturing order when required. For this,
+hit **Automate Orders**
 
-Then, click on your product to access the related product form and, on
-the "Inventory submenu", do not forget to select a supplier.
+A Reordering Rule can be temporarly deactivated for a given period by using the **Snooze** button.
 
-.. image:: strategies/strategies02.png
-    :align: center
+.. image:: strategies/reordering-rule-snooze-settings.png
+   :align: center
+   :alt:  Snooze feature options to temporarily deactivate Reordering Rules.
+
+Purchase/Manufacturing orders created by the Replenishment will have *Replenishment Report* as
+source document when created manually. If they are created automatically source document will be
+*OP/xxxxxxx* or *OP/xxxxxxx - Sxxxxxx* with the Sales Order(s) reference that triggered the rule.
+
+.. image:: strategies/rfq-source-document.png
+   :align: center
+   :alt: Look at the source panel showing where RFQ/MFG orders originated from.
 
 .. tip::
-    Don't forget to select the right product type. A consumable
-    can not be stocked and will thus not be accounted for in the stock valuation.
+   Don't forget to select the right product type. A consumable can not be stocked and will thus not
+   be accounted for in the stock valuation.
 
-Make to Order 
+Make to Order
 --------------
 
-The Make to Order configuration is available on your product form
-through your :menuselection:`Inventory --> Inventory control --> Products` (or any
-other module where products are available).
+As this configuration is recommended in particular cases, the Make to Order is a bit hidden in
```

Comment:
Replace `As this configuration is recommended in particular cases` with `Since the Make to Order (MTO) route is only recommended for customized products, the route is hidden by default.`